src/recipe.js

```diff
@@ -1,16 +1,20 @@
 class Recipe {
-  constructor(recipe) {
```

Comment:
We may need to import the ingredient data in the function that calculates that data as a parameter on line 11, lets review again tomorrow when we check the specs